src/ucp/tag/tag_send.c

```diff
@@ -78,6 +78,8 @@ static ucs_status_t ucp_tag_req_start(ucp_request_t *req, size_t count,
                   req, req->send.datatype, req->send.buffer, length, max_short,
                   rndv_rma_thresh, rndv_am_thresh, zcopy_thresh);
 
+    req->send.uct_comp.func = NULL;
+
```

Comment:
Not quite, really this PR is temporary solution. I'm going to add TODO comment to this PR and open corresponding issue versus usage of UCT completions in UCP which will be fixed separately.